packages/taquito-sapling/src/in-memory-viewing-key.ts

```diff
@@ -0,0 +1,76 @@
+import { b58cdecode, b58cencode, Prefix, prefix, validateSpendingKey } from '@taquito/utils';
```

Comment:
since `validateSpendingKey` is probably something that's quite specific to the Sapling package, could we consider moving that into a util.ts file in the Sapling package instead?

Reply:
I am not sure. All those validation functions are in the utils package with their prefixes, I thought it was a good place to add this one too.

Reply:
yeah that also makes sense. Sounds good!